fusionauth/resource_fusionauth_key.go

```diff
@@ -91,11 +96,7 @@ func createKey(_ context.Context, data *schema.ResourceData, i interface{}) diag
 	}
 
 	data.SetId(resp.Key.Id)
-	if err := data.Set("key_id", resp.Key.Id); err != nil {
-		return diag.Errorf("key.key_id: %s", err.Error())
-	}
-
-	return nil
+	return buildResourceDataFromKey(data, resp.Key)
```

Comment:
I've switched this return to build the key from the response. This is how the imported key works and I feel it is better/safer to use the actual values from the response, rather than the configuration.